apps/dashboard/app/javascript/active_jobs.js

```diff
@@ -64,11 +64,11 @@ function fetch_job_data(tr, row, options) {
 
     $.getJSON(jobDataUrl, function (data) {
       // Open this row
-      row.child(data.html_ganglia_graphs_table).show();
+	    row.child(data.html_extended_layout).show();
```

Comment:
Having just done the 'soft tabs' issue in the file editor, I'd guess that's our issue here. You're using hard tabs and this file had generally been soft tabs?